CMakeLists.txt

```diff
@@ -68,12 +68,19 @@ include_directories(${FLEDGE_INCLUDE_DIRS})
 
 # Add other include paths
 # We assume the 'freeopcua' header files are available here: 
-if (NOT EXISTS "${OPENDNP3_LIB_DIR}/cpp/libs/include")
-	message(FATAL_ERROR "OPENDNP3_LIB_DIR ${OPENDNP3_LIB_DIR}/cpp/libs/include \n"
-		"does not appear to be pointing at a valid OPENDNP3_LIB_DIR source tree")
-	return()
+# Set the base directory where the 'lib' or 'libs' directories should be located
```

Comment:
what is the background for this change ..?

Reply:
The 2.3.x tag does not work well with Ubuntu 22.04 with gcc11. There is a fix in 3.x.x release tag which has include limit. 
There is a change in CMake that by turn off static lib creation but shared lib. And also changed directory name - libs vs lib. 
We can enable static lib with a new var in Cmake.